src/dotnet/APIView/APIViewWeb/Pages/Shared/_CommentThreadPartial.cshtml

```diff
@@ -123,6 +102,22 @@
                                 <button type="submit" name="submit" value="Submit" class="btn btn-outline-secondary">Resolve</button>
                             </form>
                         }
+                        <div>
+                            @{
+                                var commentThreadNumber = (int)ViewData["CommentThreadNumber"];
+                                var commentThreadCount = (int)ViewData["CommentThreadCount"];
+                            }
+                            @if (commentThreadNumber != 1)
+                            {
+                                var previousCommentThreadAnchor = "comment-thread-" + (commentThreadNumber - 1);
+                                <a class="btn btn-outline-secondary" href="#@previousCommentThreadAnchor"><i class="fa fa-chevron-up" aria-hidden="true"></i></a>
+                            }
+                            @if (commentThreadNumber != commentThreadCount)
```

Comment:
Is this count updated when a new comment is added in between or if one is deleted?